src/views/offering/AddComputeOffering.vue

```diff
@@ -606,6 +606,9 @@ export default {
       }
     },
     isAdmin () {
+      if (!['Admin'].includes(this.$store.getters.userInfo.roletype)) {
+        this.isPublic = false
+      }
```

Comment:
`isAdmin` is a getter function and should not be setting variable values.
Also, this change does not solve root issue, ie, domain selection element is not visible to domain admin in create Compute Offering offering and isPublic is passed true in create API.
![Screenshot from 2020-06-23 16-19-04](https://user-images.githubusercontent.com/153340/85395300-88d5d800-b56d-11ea-9fa5-099cda2e94fa.png)
![Screenshot from 2020-06-23 16-19-30](https://user-images.githubusercontent.com/153340/85395312-8c695f00-b56d-11ea-9c70-fd4f9a852006.png)